### PR TITLE
Fixed links to documents from old Harvester versions

### PR DESCRIPTION
**Pull request by juanbrny**
### Summary
Updated links pointing to old 1.1 and 1.2 release documentation.
#### PR Checklist
- Is this a multi-tenancy feature/bug?
    - [ ] Yes, the relevant RBAC changes are at:
- Do we need to backport changes to the [old Rancher UI](https://github.com/rancher/u), such as RKE1?
    - [ ] Yes, the relevant PR is at:
- Are backend engineers aware of UI changes?
    - [ ] Yes, the backend owner is:

Related Issue #

### Occurred changes and/or fixed issues

Just links on lables
### Areas which could experience regressions

None

### CODE REVIEW

File: pkg/harvester/l10n/zh-hans.yaml [hunk 1]
```diff
@@ -746,7 +746,7 @@ harvester:
     dismissMessage: 忽略
     upgradeInfo:
       warning: 警告
-      doc: 在升级到新 Harvester 版本之前，你必须为集群执行所需的<a href="https://docs.harvesterhci.io/v1.0/upgrade/automatic/" target="_blank">升级前检查</a>。仅完成适用于环境的任务即可。
+      doc: 在升级到较新的 Harvester 版本之前，您必须仔细阅读<a href=“https://docs.harvesterhci.io/v1.3/upgrade/index” target=“_blank”>升级过程文档</a>。仅完成适用于环境的任务即可。
```
Comment (a110605) [inline pkg/harvester/l10n/zh-hans.yaml:749]: Please use " instead of “

**Review by bk201 — APPROVED** [review]

File: pkg/harvester/l10n/en-us.yaml [hunk 2]
```diff
@@ -753,8 +753,8 @@ harvester:
     dismissMessage: Dismiss it
     upgradeInfo:
       warning: WARNING
-      doc: Before you upgrade to the newer Harvester version, you must perform the required <a href="https://docs.harvesterhci.io/v1.0/upgrade/automatic/" target="_blank"> pre-upgrade checks </a>for your cluster. Complete only those tasks that apply to your environment.
-      tip: Failure to perform these checks may result in a failed upgrade or hitting known issues that require a manual workaround fix.
+      doc: Before you upgrade to the newer Harvester version, you must carefully review the <a href="https://docs.harvesterhci.io/v1.3/upgrade/index" target="_blank">upgrade process documentation</a>. Complete only those tasks that apply to your environment.
+      tip: Not complying with the documented procedure may result in a failed upgrade or hitting known issues that require a manual workaround fix.
```
Comment (jillian-maroket) [inline pkg/harvester/l10n/en-us.yaml:757]:
```suggestion
      doc: Read the <a href="https://docs.harvesterhci.io/v1.3/upgrade/index" target="_blank">documentation</a> before starting the upgrade process. Ensure that you complete procedures that are relevant to your environment and the version you are upgrading to.
      tip: Unmet system requirements and incorrectly performed procedures may cause complete upgrade failure and other issues that require manual workarounds.
```

File: pkg/harvester/l10n/en-us.yaml [hunk 3]
```diff
@@ -870,7 +870,7 @@ harvester:
         invalid: '"Exclude list" is invalid.'
         addIp: Add Exclude IP
       warning: 'WARNING: <br/> Any change to storage-network requires shutting down all VMs before applying this setting. <br/> Users have to ensure the Cluster Network is configured and VLAN Config will cover all nodes and ensure the network connectivity is working and expected in all nodes.'
-      tip: 'IP Range should be in IPV4 format. <code>Number of IPs Required = Number of Nodes * 4 + Number of Disks * 2 + Number of Images to Download/Upload </code>. See <a href="https://docs.harvesterhci.io/v1.2/advanced/storagenetwork#configuration-example" target="_blank">doc</a> for more details.'
+      tip: 'IP Range should be in IPV4 format. <code>Number of IPs Required = Number of Nodes * 4 + Number of Disks * 2 + Number of Images to Download/Upload </code>. See <a href="https://docs.harvesterhci.io/v1.3/advanced/storagenetwork#configuration-example" target="_blank">doc</a> for more details.'
```
Comment (jillian-maroket) [inline pkg/harvester/l10n/en-us.yaml:873]:
```suggestion
      tip: 'Specify an IP range in the IPv4 CIDR format. <code>Number of IPs Required = Number of Nodes * 4 + Number of Disks * 2 + Number of Images to Download/Upload </code>. For more information about storage network settings, see the <a href="https://docs.harvesterhci.io/v1.3/advanced/storagenetwork#configuration-example" target="_blank">documentation</a>.'
```

File: pkg/harvester/l10n/en-us.yaml [hunk 4]
```diff
@@ -941,7 +941,7 @@ harvester:
     internal:
       rancher:
         title: Access Embedded Rancher UI
-        titleDescription: We only support to use the embedded Rancher dashboard for debugging and validation purpose. For Rancher's multi-cluster and multi-tenant integration, please refer to the docs <a target="_blank" href="https://docs.harvesterhci.io/v1.2/rancher/index" rel="noopener noreferrer nofollow">here</a>.
+        titleDescription: We only support to use the embedded Rancher dashboard for debugging and validation purpose. For Rancher's multi-cluster and multi-tenant integration, please refer to the docs <a target="_blank" href="https://docs.harvesterhci.io/v1.3/rancher/rancher-integration" rel="noopener noreferrer nofollow">here</a>.
```
Comment (jillian-maroket) [inline pkg/harvester/l10n/en-us.yaml:944]:
```suggestion
        titleDescription: You can only use the embedded Rancher UI for debugging and validation purposes. For more information about how Harvester integrates with Rancher, see the <a target="_blank" href="https://docs.harvesterhci.io/v1.3/rancher/rancher-integration" rel="noopener noreferrer nofollow">documentation</a>.
```

**Review by jillian-maroket — COMMENTED** [review]
@juanbrny Thank you for contributing. Please apply the requested changes.

**Comment by a110605** [comment]
Hi @jillian-maroket , I've updated with the suggestions. Please have take a look again. 

Note. I will do cherry-pick to master and harvester-release-v1.4 once this PR merged.

**Review by jillian-maroket — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**